README.md

```diff
@@ -408,6 +410,26 @@ $ sudo -H pip2 install --upgrade gmusicapi soundcloud youtube-dl pafy pycountry
 
 &nbsp;&nbsp;
 
+# Uninstall
+
+## Debian / Ubuntu / Raspbian
+
+To uninstall Tizonia and all its dependencies, run the following commands
+
+<details><summary><b>Show details</b></summary>
+
+```bash
+
+# Remove/purge all tizonia packages installed in the system
+sudo apt-get -y remove --purge tizonia-all && sudo apt-get -y autoremove
+tizonia-dpkg-build -r
```

Comment:
This line is only needed in case one is doing local development. It removes each Debian package, by name, one by one. But this is not needed for a user that installed the official release from Bintray. In that case, removing the tizonia-all metapackage + apt autoremove, removes all of Tizonia's packages and the dependencies.